README.md

```diff
@@ -132,9 +134,15 @@ val envB = ChutneyEnvironment(
         systemBprime
     )
 )
-
 ```
 
+#### Define environments as JSON
+
+Alternatively, environments defined as JSON files could be used. The schema used follows Chutney server REST API environments export.
```

Comment:
Alternatively, you can describe your environments with JSON files.

They follow the model from Chutney Environment module : https://github.com/chutney-testing/chutney/blob/master/environment/src/main/java/com/chutneytesting/environment/api/dto/EnvironmentDto.java

In order to use environments from JSON files, you should store them in a folder named **environment**.
The default path is ```./chutney_env```, but you can override it with a constructor parameter : ```Launcher(environmentJsonRootPath = "./chutney_env")```

Reply:
done